### PR TITLE
Fixes from main

### PR DESCRIPTION
**Pull request by marisand**
Bugfixes from main.

### CODE REVIEW

File: OlinkAnalyze/R/Olink_anova.R [hunk 1]
```diff
@@ -37,7 +37,7 @@
 #'
 #' library(dplyr)
 #'
-#' npx_df <- npx_data1 %>% dplyr::filter(!grepl('control',SampleID, ignore.case = TRUE))
+#' npx_df <- npx_data1 %>% filter(!grepl('control',SampleID, ignore.case = TRUE))
```
Comment (kathy-nevola) [inline OlinkAnalyze/R/Olink_anova.R:40]: should we selectively import in examples?

File: OlinkAnalyze/R/Olink_anova.R [hunk 2]
```diff
@@ -430,15 +447,13 @@ olink_anova_posthoc <- function(df,
       message(paste("Means estimated for each assay from ANOVA model: ",formula_string))
     }
 
-
-
     anova_posthoc_results <- df %>%
       dplyr::filter(OlinkID %in% olinkid_list) %>%
       dplyr::mutate(OlinkID = factor(OlinkID, levels = olinkid_list)) %>%
       dplyr::group_by(Assay, OlinkID, UniProt, Panel) %>%
-      dplyr::do(data.frame(summary(emmeans::emmeans(lm(as.formula(formula_string),data=.),
```
Comment (kathy-nevola) [inline OlinkAnalyze/R/Olink_anova.R:439]: why was summary removed here

File: OlinkAnalyze/R/linear_mixed_model.R [hunk 3]
```diff
@@ -492,10 +492,10 @@ single_posthoc <- function(data, formula_string, effect, mean_return){
   the_model <- emmeans::emmeans(single_lmer(data, formula_string),
                                 specs=as.formula(paste0("pairwise~", paste(effect,collapse="+"))),
                                 cov.reduce = function(x) round(c(mean(x),mean(x)+sd(x)),4),
-                                lmer.df="satterthwaite")
-  the_model <- summary(the_model,infer=c(T,T),
```
Comment (kathy-nevola) [inline OlinkAnalyze/R/linear_mixed_model.R:496]: unclear why summary was removed here

File: OlinkAnalyze/R/Olink_anova.R (hunk 1 above)
Comment (marisand) [reply OlinkAnalyze/R/Olink_anova.R:40]: No, we agreed on using library(dplyr), that's why it was removed. The dplyr::filter call is currently in the file in develop and shouldn't be there.

File: OlinkAnalyze/R/Olink_anova.R (hunk 2 above)
Comment (marisand) [reply OlinkAnalyze/R/Olink_anova.R:439]: This is what caused the tukey-sidak warning

File: OlinkAnalyze/R/linear_mixed_model.R (hunk 3 above)
Comment (marisand) [reply OlinkAnalyze/R/linear_mixed_model.R:496]: See above

**Pull request merged**